src/main/resources/build.properties

```diff
@@ -6,3 +6,5 @@ springerNatureAPIKey=${springerNatureAPIKey}
 astrophysicsDataSystemAPIKey=${astrophysicsDataSystemAPIKey}
 ieeeAPIKey=${ieeeAPIKey}
 biodiversityHeritageApiKey=${biodiversityHeritageApiKey}
+Single\ instance=????
```

Comment:
Ah, that is some issue with intellij. We need to refine our guides.
 https://devdocs.jabref.org/code-howtos/localization.html

Reply:
I made a simple modification, but I don't know if I can pass the test. My knowledge is too shallow, I am new to open source, I really appreciate your help.